packages/plugin-core/src/_extension.ts

```diff
@@ -680,6 +680,8 @@ export async function _activate(
       const wsService = new WorkspaceService({ wsRoot });
       let previousWorkspaceVersionFromWSService = wsService.getMeta().version;
       if (
+        // Fix a temporary issue where CLI was writing an invalid version number to .dendron.ws:
+        !semver.valid(previousWorkspaceVersionFromWSService) ||
```

Comment:
there is a chance that previousWorkspaceVersionFromState is undefined. can we do a separate if block and just write the previous version to be equal to `currentVersion`?